doc/contributing/primordials.md

```diff
@@ -400,6 +400,7 @@ process.on('exit', () => console.log(finallyBlockExecuted)); // true
 
 You can use safe alternatives from primordials that differ slightly from the
 original methods:
+
```

Comment:
This change has already landed on `master` and is unrelated to this PR, can you please rebase?

Reply:
done.